### PR TITLE
Binance margin

### PR DESCRIPTION
**Pull request by giladwa**

### CODE REVIEW

File: python/ccxt/binance.py [hunk 1]
```diff
@@ -973,11 +1069,9 @@ def fetch_markets(self, params={}):
...
                 entry['limits']['cost']['min'] = min_notional
 
-            if leverage_limits:
-                symbol_position_limits = self.safe_value(leverage_limits, symbol)
-                market["position_limits"] = symbol_position_limits
-                max_leverage = max([leverage_limit["leverage"]["max"] for leverage_limit in symbol_position_limits])
-                entry['limits']['leverage'] = {'max': max_leverage}
+            exists = self.handle_leverage_limits(entry, leverage_limits, market, symbol)
+            if exists is False:
+                continue
```
Comment (giladwa) [inline python/ccxt/binance.py:1074]: Why are we continuing? if there's no leverage limit? what is filtered here? also from non-binance cross exchange?

Comment (orikalinski) [reply python/ccxt/binance.py:1074]: we're fetching all the markets inside the publicGetExchangeInfo for spot exchange and then we want to filter the non supported symbols for each exchange Binance Cross-Margin and Binance Isolated-Margin

Comment (giladwa) [reply python/ccxt/binance.py:1074]: ah because binance has no leverage_limits, kk

**Comment by giladwa** [comment]
@orikalinski Approved

**Pull request merged**